### PR TITLE
Shorten Text with Changes

### PR DESCRIPTION
**Pull request by Harsh-Singh-Rajput**

### CODE REVIEW

**Comment by Harsh-Singh-Rajput** [comment]
Pls check and review this

**Comment by audreyfeldroy** [comment]
Thanks so much for your thoughtful attention to the text here @Harsh-Singh-Rajput. I really like your wording improvements: they make the text sound more caring. I'll iterate a little and then merge this PR in.

File: components/HealthWorkshopCard.js [hunk 1]
```diff
@@ -25,22 +25,17 @@ function HealthWorkshopCard() {
...
-            disparities in health conditions and care such as people of color,
-            elders, young mothers including teen mothers, migrant workers,
-            low-income families, foster families, unhoused families, and
-            families of incarcerated parents.<br />
-            Your donation covers health-themed mini gift bags, raffle prizes to
-            motivate people to attend a workshop, equipment to run the workshop
-            such as trainer simulaids for edema and breast disease, costs to
-            hire public health educators to prepare presentation and take-home
-            materials, and any venue costs if not donated.
+          Helps in establishing One free health education workshop for a community that is facing major disparities in health conditions
```
Comment (audreyfeldroy) [inline components/HealthWorkshopCard.js:28]:
```suggestion
          Help in establishing one free health education workshop for a community that is facing major disparities in health conditions, 
```

File: components/HealthWorkshopCard.js [hunk 2]
```diff
@@ -25,22 +25,17 @@ function HealthWorkshopCard() {
...
-            elders, young mothers including teen mothers, migrant workers,
-            low-income families, foster families, unhoused families, and
-            families of incarcerated parents.<br />
-            Your donation covers health-themed mini gift bags, raffle prizes to
-            motivate people to attend a workshop, equipment to run the workshop
-            such as trainer simulaids for edema and breast disease, costs to
-            hire public health educators to prepare presentation and take-home
-            materials, and any venue costs if not donated.
+          Helps in establishing One free health education workshop for a community that is facing major disparities in health conditions 
+          and to help those people who need care and attention like elders, young mothers including teen mothers, migrant workers,
```
Comment (audreyfeldroy) [inline components/HealthWorkshopCard.js:29]:
```suggestion
          and to help those who need extra care and attention.
```

File: components/HealthWorkshopCard.js [hunk 3]
```diff
@@ -25,22 +25,17 @@ function HealthWorkshopCard() {
...
-            low-income families, foster families, unhoused families, and
-            families of incarcerated parents.<br />
-            Your donation covers health-themed mini gift bags, raffle prizes to
-            motivate people to attend a workshop, equipment to run the workshop
-            such as trainer simulaids for edema and breast disease, costs to
-            hire public health educators to prepare presentation and take-home
-            materials, and any venue costs if not donated.
+          Helps in establishing One free health education workshop for a community that is facing major disparities in health conditions 
+          and to help those people who need care and attention like elders, young mothers including teen mothers, migrant workers, 
+          low-income families, foster families, unhoused families, and families of incarcerated parents.<br />
```
Comment (audreyfeldroy) [inline components/HealthWorkshopCard.js:30]:
```suggestion
          <br />
```

File: components/HealthWorkshopCard.js [hunk 4]
```diff
@@ -25,22 +25,17 @@ function HealthWorkshopCard() {
...
+          Also, Your donation will be put in service for health-themed mini gift bags, raffle prizes to motivate people to attend a workshop,
```
Comment (audreyfeldroy) [inline components/HealthWorkshopCard.js:31]:
```suggestion
          Also, your donation will be put in service for health-themed mini gift bags, raffle prizes to motivate people to attend a workshop, 
```

File: components/HealthWorkshopCard.js [hunk 5]
```diff
@@ -25,22 +25,17 @@ function HealthWorkshopCard() {
...
+          low-income families, foster families, unhoused families, and families of incarcerated parents.<br />
+          Also, Your donation will be put in service for health-themed mini gift bags, raffle prizes to motivate people to attend a workshop, 
+          equipment to run the workshop such as trainer simulaids for edema and breast disease, 
+          and to hire public health educators to prepare presentations...
           </p>
           <p>
           <form onSubmit={handleSubmit}>
               <button className="btn btn-primary">
-              Produce 1 health workshop for $5000 <span />
+              Help in Creating One health workshop for $5000 <span />
```
Comment (audreyfeldroy) [inline components/HealthWorkshopCard.js:38]:
```suggestion
              Help in Creating One Health Workshop for $5000 <span />
```

**Review by audreyfeldroy — APPROVED** [review]
Small iterations on @Harsh-Singh-Rajput's good work.

**Comment by audreyfeldroy** [comment]
Looks like there's a conflict in `components/HealthWorkshopCard.js` because I merged in #56 which reformatted the file for coding style.

I'm getting quite sleepy. @Harsh-Singh-Rajput or anyone else, if you're up to it, try resolving the conflicts. Otherwise I'm happy to do it tomorrow.

**Comment by Harsh-Singh-Rajput** [comment]
> Thanks so much for your thoughtful attention to the text here @Harsh-Singh-Rajput. I really like your wording improvements: they make the text sound more caring. I'll iterate a little and then merge this PR in.

no problem

**Comment by Harsh-Singh-Rajput** [comment]
> Looks like there's a conflict in `components/HealthWorkshopCard.js` because I merged in #56 which reformatted the file for coding style.
> 
> I'm getting quite sleepy. @Harsh-Singh-Rajput or anyone else, if you're up to it, try resolving the conflicts. Otherwise I'm happy to do it tomorrow.

didn't understand what the problem 😢 could you solve it and  tell me later

**Comment by audreyfeldroy** [comment]
Of course, don't worry! You did great with your contribution and there's no need to do anything further on your end.

If you're curious to learn more:

* A merge conflict is when 2 people edit the same file in different ways. Here, someone reformatted the file you were working on with line breaks, then submitted a PR that got merged before yours. It happens all the time and is very easy to fix.

* GitHub has a very cool interactive editor for resolving merge conflicts. [More info here.](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) When you click the "Resolve conflicts" button, it opens the interactive editor. It's fun to try out.

You're welcome to experiment with the "Resolve conflicts" button if you'd like. This is a safe space to make mistakes and learn. You can click it and see what it does, and play around. 

No pressure, though! If I see the "Conflicting files" message when I check back later, I'll go ahead and handle this part 🙂

**Comment by audreyfeldroy** [comment]
I've merged this. Thanks again @Harsh-Singh-Rajput!

**Comment by Harsh-Singh-Rajput** [comment]
> Of course, don't worry! You did great with your contribution and there's no need to do anything further on your end.
> 
> If you're curious to learn more:
> 
> * A merge conflict is when 2 people edit the same file in different ways. Here, someone reformatted the file you were working on with line breaks, then submitted a PR that got merged before yours. It happens all the time and is very easy to fix.
> * GitHub has a very cool interactive editor for resolving merge conflicts. [More info here.](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) When you click the "Resolve conflicts" button, it opens the interactive editor. It's fun to try out.
> 
> You're welcome to experiment with the "Resolve conflicts" button if you'd like. This is a safe space to make mistakes and learn. You can click it and see what it does, and play around.
> 
> No pressure, though! If I see the "Conflicting files" message when I check back later, I'll go ahead and handle this part 🙂

thanks for the information

**Pull request merged**